tests/unit/OptionsListUtilsTest.js

```diff
@@ -1928,9 +1928,54 @@ describe('OptionsListUtils', () => {
                 name: 'Taxi',
             },
         ];
+        const categoriesIncorrectOrdering3 = {
+            'Movies: Mr. Nobody': {
+                enabled: true,
+                name: 'Movies: Mr. Nobody',
+            },
+            Movies: {
+                enabled: true,
+                name: 'Movies',
+            },
+            'Dr. House': {
```

Comment:
Uber-nit: "House, M.D."

Reply:
What does "Uber-nit" mean?

Reply:
I've added that category - e70e40b.

Reply:
Well, a "nit" is when you nit-pick, i.e., give attention in code review to small details that don't really matter a lot... 

"uber-nit" is when this is taken to an extreme. It's not a standard term

I just meant that in English, it's "House, M.D." (House, Medical Doctor), while "Dr. House" might be correct in other languages.

Reply:
Ah, okay, Let's keep both categories, make our tests stricter 🙂